WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/ReattachUploadingMediaUseCase.kt

```diff
@@ -0,0 +1,41 @@
+package org.wordpress.android.ui.posts.editor.media
+
+import org.wordpress.android.editor.EditorMediaUploadListener
+import org.wordpress.android.fluxc.model.MediaModel
+import org.wordpress.android.ui.posts.EditPostRepository
+import org.wordpress.android.ui.uploads.UploadServiceFacade
+import javax.inject.Inject
+
+class ReattachUploadingMediaUseCase @Inject constructor(
+    private val uploadServiceFacade: UploadServiceFacade
+) {
+    fun reattachUploadingMediaForAztec(
+        editPostRepository: EditPostRepository,
+        isAztec: Boolean,
```

Comment:
I'm thinking about this approach and it just doesn't feel right - passing a boolean field and not do anything if it's false. I'd move this check to the EditorMedia and maybe renamed this use case to include `Aztec` so no one is tempted to use it with GB?

Reply:
Yeah I agree it feels weird. I wanted to emphasise it can be removed as soon as we remove Aztec,  but it doesn't make sense. 
I've moved it to EditorMedia as you suggested. We are still passing a boolean and do nothing when it's false though. Didn't you want me to move it to the EditPostActivity?

Reply:
I somehow prefer the `if` being in the `EditPostActivity` because it's one less method parameter and it makes the intent clear since it will be obvious that `mEditorMedia.reattachUploadingMediaForAztec` shouldn't be called unless the current editor is Aztec. It's a small change though, so this can still be pushed as is.